### DISCUSSION TITLE
Error correction and assembling warnings

### DISCUSSION THREAD

**Discussion by liuyca1** [ablab/spades#1246] [Q&A]
### Description of bug

hello, I get some errors while running spades v3.13.1, I want to assemble a 50M fungi genome by Illumina and Pacbio sequence.
[spades.log](https://github.com/ablab/spades/files/14320682/spades.log)

error:
=== Error correction and assembling warnings:
 * 0:53:34.294     9G / 208G  WARN   ScaffoldingUniqueEdgeAna (scaff_supplementary.cpp   :  59)   Less than half of genome in unique edges!
======= Warnings saved to /public/home/liuyuanchao/Russula_genome/asembly20240205/warnings.log

[warnings.log](https://github.com/ablab/spades/files/14320695/warnings.log)

### spades.log

[spades.log](https://github.com/ablab/spades/files/14320691/spades.log)

### params.txt

[params.txt](https://github.com/ablab/spades/files/14320693/params.txt)

### SPAdes version

SPAdes version: 3.13.1

### Operating System

Linux-3.10.0-957.el7.x86_64-x86_64-with-glibc2.17

### Python Version

Python version: 3.9.7

### Method of SPAdes installation

manual

### No errors reported in spades.log

- [X] Yes

**Comment by asl** [comment]
Given the genome in question it's ok

**Reply by liuyca1** [reply]
Confused, something wrong with the genome?

**Reply by asl** [reply]
It's fungal diploid genome with lots of repeats. So it's ok to have more than half of genomes in repeats.
